notebooks/official/ml_metadata/vertex-pipelines-ml-metadata.ipynb

```diff
@@ -0,0 +1,1179 @@
+{
```

Comment:
<p>for packages_to_install, you don't use joblib and sklearn in this component.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>